docs/user_guide/index.rst

```diff
@@ -8929,6 +9144,103 @@ and this BIE is reused by another BIE. In this case, Score displays a
 notification at the bottom of the page and keeps a more detail log which
 is accessible via the notification page.
 
+Multi-tenant management
+=======================
+
+Multi-tenant mode
+-----------------
+Score multi-tenant feature enables OAGi to offer Score-based BIE-development services to 
+multiple members on a single instance with visibility of BIEs managed by member (i.e. tenant)
+
+Enable multi-tenant mode
+~~~~~~~~~~~~~~~~~~~~~~~~
+By default Score does not use multi-tenant mode. To enable multi-tenant mode:
+
+1. Click on your username (top-right) | "Settings" | "Application settings".
+2. Under the Multi-tenant mode label click the "Enable" button.
+
```

Comment:
Should we also say something like this? Multi-tenant mode should be enabled at the first deployment of Score. And while it is possible to switch on or off the multi-tenant mode, it is not recommended to go back and forth between the two modes.